pkg/scheduler/framework/types.go

```diff
@@ -904,6 +904,25 @@ func (n *NodeInfo) RemoveNode() {
 	n.Generation = nextGeneration()
 }
 
+// GetNodes get v1.Node slice from a NodeInfo slice
+func GetNodes(infos []*NodeInfo) []*v1.Node {
+	res := make([]*v1.Node, len(infos))
+	for i := 0; i < len(infos); i++ {
+		res[i] = infos[i].node
+	}
+	return res
+}
+
+// GetNodeInfos  get NodeInfo slice from a v1.Node slice
+func GetNodeInfos(nodes []*v1.Node) []*NodeInfo {
```

Comment:
```suggestion
func BuildNodeInfos(nodes []*v1.Node) []*NodeInfo {
```

But also, this is not supposed to be used in production code, so put it in the `pkg/scheduler/testing` package.